README.Rmd

```diff
@@ -16,7 +16,7 @@ knitr::opts_chunk$set(
   comment = "#>",
   fig.path = "README-"
 )
-library(blob)
+pkgload::load_all(".")
```

Comment:
This needs to remain as `library(blob)`, but needs to occur in the `example` chunk so that we're providing readers with "real" code